### PR TITLE
#2549. Update built-in identifier tests. Add missing ones. Part 1

### PR DESCRIPTION
**Pull request by sgrekhov**
Before adding new tests for the new buil-in identifiers `out` and `inout` let's reorganize existing tests and add the missing ones. Some tests are shown as deleted/added, but in fact all deleted tests just renamed.

### CODE REVIEW

File: Language/Expressions/Identifier_Reference/built_in_identifier_A02_t02.dart [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+/// `as` is used as the declared name of a type variable.
+/// @author ngl@unipro.ru
+
+class C<as> {
+//      ^^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+mixin M<abstract> {
```
Comment (eernstg) [inline Language/Expressions/Identifier_Reference/built_in_identifier_A02_t02.dart:19]:
```suggestion
mixin M<as> {
```

Comment (eernstg) [reply Language/Expressions/Identifier_Reference/built_in_identifier_A02_t02.dart:19]: Similarly for a few more locations below.

**Review by eernstg — COMMENTED** [review]
Looks good! Just one library had a couple of typos.

I assume there will be another PR containing similar tests for `Function | get
| implements | import | interface | late | library | mixin | operator
| part | required | set | static | typedef`?

**Comment by sgrekhov** [comment]
Thank you. Fixed.
Yes, there will be another PR with the other built-in identifiers. Working on it now

**Review by eernstg — APPROVED** [review]
LGTM

**Comment by eernstg** [comment]
Sounds good, thanks!

**Pull request merged**